packages/common/src/services/sdk/storageNodeSelector.ts

```diff
@@ -8,6 +8,8 @@ import { Env } from '../env'
 import { getBootstrapNodes } from './bootstrapNodes'
 import { DiscoveryNodeSelectorService } from './discovery-node-selector'
 
+const logger = new Logger()
```

Comment:
kewl

Reply:
Shouldn't be necessary in latest SDK! :)